introduction/data-model.md

```diff
@@ -56,30 +56,25 @@ Using our example above, this approach would result in 9 different "time series"
 ```
 The number of such time series scales with the cross-product of the
 cardinality of each tag, i.e., (# names) &times; (# device ids) &times; (#
-location ids) &times; (device types).
+location ids) &times; (device types). Some time-series databases struggle as 
+cardinality increases, ultimately limiting the number of device types and devices 
+you can store in a single database.
 
-And each of these "time series" then has its own set of time/value sequences.
+TimescaleDB supports narrow models and does not suffer from the same cardinality limitations 
+as other time-series databases do. A narrow model makes sense if you collect each metric 
+independently. It allows you to add new metrics as you go by adding a new tag without 
+requiring a formal schema change.
 
-Now, this approach may make sense if you collect each of your metrics
-independently, with little to no metadata.
-
-But in general, we believe that this approach is limiting. It loses the
-inherent structure in the data, making it
-harder to ask a variety of useful questions. For example:
-- What was the state of the system when `free_mem` went to 0?
-- How does `cpu_1m_avg` correlate with `free_mem`?
-- What is the average `temperature` by `location_id`?
-
-We also find this approach cognitively confusing. Are we really collecting
-9 different time-series, or just one collection of data with a variety
-of metadata and metrics readings?
+However, a narrow model is not as performant if you are collecting many metrics with the 
+same timestamp, since it requires writing a timestamp for each metric. This ultimately 
+results in higher storage and ingest requirements. Further, if you typically query multiple 
+metrics together, it is faster to store them in a wide table format, which we will cover 
+in the following section. 
```

Comment:
Also queries are more complex if you want to correlate different values. Every further value you want to correlate requires another JOIN

Reply:
Addressed